bindings/wasm/examples/model-viewer.html

```diff
@@ -59,25 +64,26 @@
         metalness: 1,
         roughness: 0.2
       }));
-
-      const icosahedron = simplify(new THREE.IcosahedronGeometry(0.16));
-      const manifold_1 = Module.cube(0.2, 0.2, 0.2, true);
-      // const manifold_1 = Module
-      //   .revolve([[[0.1, 0.1], [0.2, 0.1], [0.1, 0.2]]], 64)
-      //   .translate(0, 0, -0.1);
-      const manifold_2 = new Module.Manifold(geometry2mesh(icosahedron));
-
-      const csg = function (operation) {
-        mesh.geometry?.dispose();
-        mesh.geometry = mesh2geometry(Module[operation](manifold_1, manifold_2).getMesh());
-        push2MV(mesh);
-      };
-
-      document.querySelector('select').onchange = function (event) {
-        csg(event.target.value);
-      };
-
-      csg('difference');
+      document.querySelector('#compile').onclick = function (e) {
```

Comment:
Can we copy this into a new html page instead of editing the MV example? I'd like to keep that as a minimal example page, since this editor will probably grow a bit in complexity.